src/utilities/theme-utils.ts

```diff
@@ -31,6 +33,11 @@ export const setThemeColors = (primary: string, secondary: string) => {
   document.documentElement.style.setProperty("--theme-secondary-text-color", getContrastYIQColor(secondary));
 };
 
+export const setBackgroundImage = (imageUrl:string) => {
+  const el = document.querySelector(".app") as HTMLElement;
+  el && el.style.setProperty("background-image",`url("${imageUrl}")`);
+};
+
```

Comment:
One note here.  This function doesn't really belong in the theme-utils module since it isn't related to themes.  I would put it in activity-utils or, if you think we might need more similar content, make an image-utils module.

Reply:
Moved it to activity-utils.